### PR TITLE
Don't concatenate all.css

### PR DESCRIPTION
**Pull request by piuccio**
When merging together all files, exclude the output file to prevent it from growing out of control.

If you have a look at `all.css` right now it contains duplicate definition for some classes.

### CODE REVIEW

**Comment by cutandpastey** [comment]
Wow, thats a massive fail on my part. Thanks for looking into this. Im wondering if instead of a filter, It would be better to just delete the file at the beginning of a build?

**Comment by piuccio** [comment]
That's a great idea. The filter was easier to implement, I'm a bit lost with the ES7 syntax. Let me try

**Comment by piuccio** [comment]
This should do the job

File: bin/lib/release.js [hunk 1]
```diff
@@ -56,6 +57,10 @@ export default (releaseType) => {
         throw new Error('No .gitignore file');
       }
 
+      let outputFile = path.resolve(__dirname, `../../styles/${OUTPUT_CSS_FILENAME}`);
+      console.log(`- Deleting output file ${outputFile}`.green);
+      yield pUnlink(outputFile);
```
Comment (cutandpastey) [inline bin/lib/release.js]: What happens if this file does note exist? DO we get an error? If so, you can use `fs.existsSync` to check it.

Comment (piuccio) [inline bin/lib/release.js]: Yes we get an error. Exists sync looks so horrible in that code base, but if it's fine for you...

Comment (cutandpastey) [inline bin/lib/release.js]: Yeah, sadly fs.exists returns a `bool` not `err, bool` so im not 100% sure what happens when you `denodeify` it see: https://nodejs.org/api/fs.html#fs_fs_exists_path_callback so I guess we have to use it :(

:+1:  for code !

Comment (theefer) [inline bin/lib/release.js]: > Yeah, sadly fs.exists returns a bool not err, bool

lol wtf :poop:

Comment (cutandpastey) [inline bin/lib/release.js]: Yeah, sucks :(

**Pull request merged**